spec/requests/users_controller_spec.rb

```diff
@@ -1026,7 +1026,7 @@
               it "allows changing email, dpo, key_contact" do
                 patch "/users/#{other_user.id}", headers: headers, params: params
                 other_user.reload
-                expect(other_user.email).to eq(new_email)
+                expect(other_user.unconfirmed_email).to eq(new_email)
```

Comment:
Do we set email to == unconfirmed_email after it's been confirmed? Is that tested? Also, do we set unconfirmed email to nil after it's been confirmed?

Reply:
Devise does yes. This only applies when you're "reconfirming", i.e. if you've changed your email address, it doesn't get used on first creation.

Reply:
https://github.com/heartcombo/devise/blob/f8d1ea90bc328012f178b8a6616a89b73f2546a4/lib/devise/models/confirmable.rb#L91